sentences/fi/_common.yaml

```
@@ -0,0 +1,61 @@
+language: "fi"
+responses:
+  errors:
+    no_intent: "Anteeksi, en ymmärtänyt"
+    no_area: "Ei aluetta nimeltä {{ area }}"
+    no_domain: "Alueella {{ area }} ei ole {{ domain }}"
+    no_device_class: "Alue {{ area }} ei sisällä {{ device_class }}"
+    no_entity: "Ei laitetta nimeltä {{ entity }} löytynyt"
+    handle_error: "Tuntematon virhe tapahtui käsittelyssä"
+
+lists:
+  color:
+    values:
+      - in: "valkoinen"
+        out: "white"
+      - in: "musta"
+        out: "black"
+      - in: "punainen"
+        out: "red"
+      - in: "oranssi"
+        out: "orange"
+      - in: "keltainen"
+        out: "yellow"
+      - in: "vihreä"
+        out: "green"
+      - in: "sininen"
+        out: "blue"
+      - in: "pinkki"
+        out: "purple"
+      - in: "ruskea"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+  expansion_rules:
+    name: "{name}"
+    area: "{area}"
+    what_is: "(mikä on | mikä se on | mitä varten se on | mikä)"
+    is_it: "(onko | onko se)"
+    brightness: "{brightness} [percent]"
+    turn_on: "(pane | päälle | käynnistä)"
+    turn_off: "(sammuta | sammuttaa | lopeta)"
+    temp: "(lämpötila)"
+    temperature: "{temperature} [astetta] [{temperature_unit}]"
+  skip_words:
+    - "Ole kiltti"
```

Indentation is wrong here. Makes expansion rules and skip words part of the lists.
```suggestion
expansion_rules:
  name: "{name}"
  area: "{area}"
  what_is: "(mikä on | mikä se on | mitä varten se on | mikä)"
  is_it: "(onko | onko se)"
  brightness: "{brightness} [percent]"
  turn_on: "(pane | päälle | käynnistä)"
  turn_off: "(sammuta | sammuttaa | lopeta)"
  temp: "(lämpötila)"
  temperature: "{temperature} [astetta] [{temperature_unit}]"
skip_words:
  - "Ole kiltti"
```